test/new-e2e/tests/installer/host/host.go

```diff
@@ -215,8 +215,11 @@ func (h *Host) WaitForFileExists(useSudo bool, filePaths ...string) {
 // This is because of a race condition where the trace agent is not ready to receive traces and we send them
 // meaning that the traces are lost
 func (h *Host) WaitForTraceAgentSocketReady() {
-	_, err := h.remote.Execute("timeout=30; while ! grep -q 'Listening for traces at unix://' <(sudo journalctl _PID=`systemctl show -p MainPID datadog-agent-trace | cut -d\"=\" -f2`); do sleep 1; ((timeout--)); done; [ $timeout -ne 0 ]")
-	require.NoError(h.t(), err, "trace agent did not become ready")
+	require.EventuallyWithT(h.t(), func(t *assert.CollectT) {
+		// this endpoint is no-op but it will fail if the trace agent is not ready
+		_, err := h.remote.Execute("curl -XGET --unix-socket /var/run/datadog/apm.socket http:/services")
```

Comment:
Are we missing a / ?
```suggestion
		_, err := h.remote.Execute("curl -XGET --unix-socket /var/run/datadog/apm.socket http://services")
```

Reply:
The current version definitely works since it ran properly in CI
Also I found various posts using a single slash, eg. https://superuser.com/a/925610/1205764

I searched around but couldn't find a clear answer on the difference, I think both are pretty much equivalent now
I can give the extra slash a try if you want 🤷‍♂️